server/models/competition_instance.js

```diff
@@ -70,16 +71,18 @@ class CompetitionInstance {
     return this.hasStarted() ? this.comp.getWinners() : null;
   }
 
-  _createNewGame(gameInfo) {
+  _createNewGame(gameInfo, lastGame) {
     if (gameInfo) {
-      let gameId = this.gameEngine.create(gameInfo.gameParams);
+      let lastGameState = lastGame ? { lastGame: _.omit(lastGame.game, 'params') } : {};
+      let gameParams = Object.assign({}, gameInfo.gameParams, lastGameState);
```

Comment:
For building objects this way I've been using [Object Spread Initializers](https://github.com/sebmarkbage/ecmascript-rest-spread/blob/master/Spread.md), which I activated in babel despite being ES7. Can you use it here too for the sake of consistency?